Makefile

```diff
@@ -28,7 +28,7 @@ test_lint:
 	}
 
 test_go:
-	go test -race -p=1 ./...
+	go test -race -p=1 -short ./...
```

Comment:
+1 for faster tests! But I see below that we now `Skip` a lot of short tests, too. Ist that on purpose?